kedro_neptune/config.py

```diff
@@ -38,11 +38,8 @@ class NeptuneConfig:
 
 
 def get_neptune_config() -> NeptuneConfig:
-    session: KedroSession = get_current_session()
-    context = session.load_context()
-    # pylint: disable=protected-access
-    credentials = context._get_config_credentials()
-    config = context.config_loader.get('neptune**')
+    credentials = ConfigLoader('conf', 'local').get("credentials_neptune*")
```

Comment:
I'm still not sure about hardcoding this paths

Reply:
sorry, I missed the first time you weren't sure ;) I'll look into it